buildenv/jenkins/common/build.groovy

```diff
@@ -326,8 +326,63 @@ def archive_sdk() {
     }
 }
 
+/*
+* When crash occurs in java, there will be an output in the log IEATDUMP success for DSN='JENKINS.JVM.JENKIN*.*.*.X&DS'
+* The X&DS is a macro that is passed to the IEATDUMP service on z/OS
+* The dump will be split into multiple files by the macro if the address space is large
```

Comment:
This reads better if `will be` is replaced with `is`. Similarly in the last sentence.

Reply:
Missed fixing "will be" on this line.

Reply:
poke, see previous comment

Reply:
Hey @pshipton! It was just fixed. Thanks!